packages/ens-utils/src/index.ts

```diff
@@ -13,3 +13,13 @@ export * from "./time";
 export * from "./transaction";
 export * from "./domain";
 export * from "./registration";
+
+declare global {
```

Comment:
Why are we adding this to `ens-utils`?

I don't see why `ens-utils` should do this. My current understanding is that it shouldn't. Why should anyone using `ens-utils` require that the global `toJSON` is now manipulated for everyone? That seems like a bad abstraction.

Reply:
@lightwalker-eth one reason that motivates this addition is that this library exports multiple `bigint` typed properties.

Another reason is that, any project that includes one of these objects that have `bigint` type, might face the error: 

`TypeError: Do not know how to serialize a BigInt`

And, last but not least, this issue was identified when using these properties in Storybook stories, where we have the usage of `.js` configuration files. Declaring a different `Bigint` prototype requires the usage of Typescript which would increase complexity of `storybook.namekit.io` and wouldn't make `bigint` serialization problem solved for all applications that reuse the `bigint` related utilities included in `ens-utils`...

Reply:
Please see the discussion we had on this previously.

Inside `ens-utils` we don't have any logic that performs this serialization.

If you put logic like this inside `ens-utils`, it corrupts the global namespace for every other app that integrates with `ens-utils`. If an app needs logic to serialize bigints like this, it is the responsibility of that app to add that logic themselves. We shouldn't do it as a package.

Storybook is an app, not a package. So it's ok to include special serialization logic there if it is actually needed.